packages/react-next/src/components/Pivot/Pivot.types.ts

```diff
@@ -112,10 +112,9 @@ export interface IPivotProps extends React.HTMLAttributes<HTMLDivElement> {
  */
 export type IPivotStyleProps = Required<Pick<IPivotProps, 'theme'>> &
   Pick<IPivotProps, 'className'> & {
-    /** Indicates whether Pivot has large format. */
-    rootIsLarge?: boolean;
-    /** Indicates whether Pivot has tabbed format. */
-    rootIsTabs?: boolean;
+    linkSize?: PivotLinkSizeType;
+    linkFormat?: PivotLinkFormatType;
+
```

Comment:
Instead of having them declared here again, since you already have them in `IPivotProps` you could do:

```ts
export type IPivotStyleProps = Required<Pick<IPivotProps, 'theme'>> &
     Pick<IPivotProps, 'className', 'linkFormat', 'linkSize'> & {
```